src/sentry/api/endpoints/organization_member_team_details.py

```diff
@@ -56,11 +60,16 @@ def _can_create_team_member(self, request: Request, organization, team_slug):
 
         return (
             is_active_superuser(request)
-            or self._can_admin_team(request, organization, team_slug)
+            or self._can_admin_team(request, team)
             or organization.flags.allow_joinleave
```

Comment:
Could `is_active_superuser(request) or organization.flags.allow_joinleave` be replaced with `request.access.has_global_access`?

Reply:
Yes, nice catch.